### DISCUSSION TITLE
Survey Form Using Events in javascript

### DISCUSSION THREAD

**Discussion by Abanou6** [community/community#140456] [Programming Help]
### Select Topic Area

Question

### Body

<!DOCTYPE html>
<html>
<head>
  <title>Luxury Brand Feedback</title>
</head>
<body>
  <h1>Welcome to Luxury Brand Products Feedback</h1>
  <h2>Provide Your Feedback</h2>
  <label for="name">Name:</label><br>
  <input type="text" id="name" placeholder="Your Name" required><br>
  <label for="age">Age:</label><br>
  <input type="number" id="age" placeholder="Your Age" required><br>
  <label for="email">Email:</label><br>
  <input type="email" id="email" placeholder="Your Email" required><br>
  <label for="job">Job:</label><br>
  <input type="text" id="job" placeholder="Your Job" required><br>
  <label for="designation">Designation:</label><br>
  <input type="text" id="designation" placeholder="Your Designation" required><br>
  <label for="productType">Product Type:</label><br>
  <select id="productType" required>
    <option value="perfume">Perfume</option>
    <option value="cream">Cream</option>
    <option value="oils">Oils</option>
  </select><br>
  <label for="feedbackText">Feedback:</label><br>
  <textarea id="feedbackText" rows="5" cols="30" placeholder="Enter your feedback here" required></textarea><br>
  <button id="submitBtn">Submit Feedback</button>
<hr>
<div id="userInfo" style="display: none;">
  <h2>User Information:</h2>
  <p>Name: <span id="userName"></span></p>
  <p>Age: <span id="userAge"></span></p>
  <p>Email: <span id="userEmail"></span></p>
  <p>Job: <span id="userJob"></span></p>
  <p>Designation: <span id="userDesignation"></span></p>
  <p>Feedback for  <span id="userProductChoice"></span>: <span id="userFeedback"></span></p>
</div>
<script src="./product_survey.js"></script>
</body>
</html>

------------------------------------------------------------------------------------------------------------------------------------------------
function submitFeedback(){
    alert('Thank you for your valuable feedback');
    document.getElementById('userInfo').style.display = 'block';
    document.getElementById('userName').innerHTML = username;
    document.getElementById('userAge').innerHTML = age;
    document.getElementById('userEmail').innerHTML = email;
    document.getElementById('userJob').innerHTML = job;
    document.getElementById('userDesignation').innerHTML = designation;
    document.getElementById('userProductChoice').innerHTML = productType;
    document.getElementById('userFeedback').innerHTML = feedbackText;
}
const username = document.getElementById('name').Value;
const age = document.getElementById('age').Value;
const email = document.getElementById('email').Value;
const job = document.getElementById('job').Value;
const designation = document.getElementById('designation').Value;
const productType = document.getElementById('productType').Value;
const feedbackText = document.getElementById('feedbackText').Value;
const submitButton = document.getElementById('submitBtn');
submitButton.onclick = submitFeedback;
document.addEventListener('keydown', function(event){
    if (event.key==='Enter'){
        submitFeedback();
    }
});

------------------------------------------------------------------------------------------------------------------------------------------------
* Dear colleagues, appreciate your help and support regarding the issues of the above code, because this is what shows up when executed, Please look at the below image for more info.
![image](https://github.com/user-attachments/assets/9b747aa0-2d54-422b-9703-07ee2523d32d)

**Comment by LiteBrite82** [comment]
Hi @Abanou6

Thanks for posting! Can you please add a small paragraph explaining what feedback you would like. Adding a repo link with your test code could be super helpful as well.

**Reply by Abanou6** [reply]
Hi@LiteBrite82
Thank you so much for your support and prompt reply.

Here is the link to the repo https://github.com/Abanou6/controlFlow

Actually this code sample is supposed to collect data from a client and submiting it and the second part of the page shows up the data that has already been submitted.

**the issue is that the fields - that supposed to show the submitted data- shows as undifinded.

**Comment by ebndev** [comment]
Hey @Abanou6, thanks for being a part of the GitHub Community!

Looks like your issue may be resolved by changing `.Value` to lowercase `.value` 😄
